cmd/tempo/app/modules.go

```diff
@@ -154,27 +153,32 @@ func (t *App) initQuerier() (services.Service, error) {
 	return t.querier, t.querier.CreateAndRegisterWorker(t.server.HTTP)
 }
 
+type cortexLimits struct{}
+
+func (cortexLimits) MaxQueriersPerUser(user string) int { return 0 }
+
 func (t *App) initQueryFrontend() (services.Service, error) {
 	var err error
-	t.frontend, err = cortex_frontend.New(t.cfg.Frontend.Config, util.Logger, prometheus.DefaultRegisterer)
+	t.frontend, err = cortex_frontend.New(t.cfg.Frontend.Config.FrontendV1, cortexLimits{}, util.Logger, prometheus.DefaultRegisterer)
 	if err != nil {
 		return nil, err
 	}
 
 	// custom tripperware that splits requests
-	tripperware, err := frontend.NewTripperware(t.cfg.Frontend, util.Logger, prometheus.DefaultRegisterer)
+	/*tripperware, err := frontend.NewTripperware(t.cfg.Frontend, util.Logger, prometheus.DefaultRegisterer)
```

Comment:
Is this causing conflicts with the upstream code? We need this tripperware to shard queries

Reply:
Yeah this part was still WIP.  See updates.